### PR TITLE
Posts

### PR DESCRIPTION
**Pull request by Andrii-Kozak**
Знімок екрану з 2021-02-27 19-02-43
image

### CODE REVIEW

File: app/views/posts/_post.html.slim [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.shadow
+  .card-body
+    hr
+    ul.list-inline.mt-3
+      li.list-inline-item
+        i.fas.fa-user
+        |  Mukola Kaban
```
Comment (pereslop) [inline app/views/posts/_post.html.slim:7]: = post.creator.full name
and this should be the link to creator page

File: app/views/posts/_post.html.slim [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      li.list-inline-item
+        i.far.fa-clock
+        |  February 27, 2021
```
Comment (pereslop) [inline app/views/posts/_post.html.slim:10]: post.created_at
please read about https://apidock.com/rails/ActionView/Helpers/DateHelper/time_ago_in_words, https://apidock.com/rails/ActionView/Helpers/DateHelper/distance_of_time_in_words and chose one of them

File: app/views/users/show.html.slim [hunk 3]
```diff
@@ -29,11 +29,11 @@
...
+        .card.mt-3[style="min-height: 339px;"]
           h2.link-user-show.mb-2.mt-2
             | My Friends
 
       .col-md-8
-        .card.mb-3[style="min-height: 1020px;"]
-          .text-center
-            | some news
+        .card[style="min-height: 1020px;"]
+          - @user.posts.each do |post|
```
Comment (pereslop) [inline app/views/users/show.html.slim:38]: https://stackoverflow.com/questions/12188381/render-partial-collection-array-specify-variable-name

File: spec/factories/posts.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+FactoryBot.define do
+  factory :post do
+    body { Faker::Lorem.paragraph }
+    association :creator, factory: :user
+    association :postable, factory: :group
+
+    trait :with_image do
+      image { File.open("#{Rails.root}/app/assets/images/dog.jpg") }
+    end
+    trait :for_group do
```
Comment (pereslop) [inline spec/factories/posts.rb:10]: add empty lines between traits

**Review by pereslop — COMMENTED** [review]
please fix comments

**Review by Andrii-Kozak — COMMENTED** [review]
corrected all comments

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -6,7 +6,9 @@ def index
     @users = User.ordered_by_created_at.page(params[:page])
   end
 
-  def show; end
+  def show
+    @posts = @user.posts.includes(:creator)
```
Comment (pereslop) [inline app/controllers/users_controller.rb:10]: move this to scope in post model, spope :with_creator,  -> includes(:creator)

Comment (Andrii-Kozak) [reply app/controllers/users_controller.rb]: done!

**Pull request merged**